ranger-zookeeper/src/test/java/io/appform/ranger/zookeeper/servicehub/ServiceHubTest.java

```diff
@@ -113,10 +99,22 @@ public void testHub() {
                 .withCuratorFramework(curatorFramework)
                 .build();
         provider1.start();
-
         refreshProviderSignal.trigger();
+
+        ExternalTriggeredSignal<Void> refreshHubSignal = new ExternalTriggeredSignal<>(() -> null, Collections.emptyList());
```

Comment:
ExternalTriggeredSignal<Void> -> val

Reply:
Updated

Reply:
Did you run tests after this? This one is going to be bound to .withExtraRefreshSignal, and is the compiler not throwing any type issues after val on this?

Reply:
You are right to fix it I made a change for it, added void type for the signal. 
And then ran the tests and all of em worked.

Reply:
Cool, yeah. That makes sense!